packages/api/tests/test-rules-model.js

```diff
@@ -117,4 +119,29 @@ test('update a kinesis type rule value, resulting in new arn', async (t) => {
   t.is(newRule.name, rule.name);
   t.not(newRule.rule.vale, rule.rule.value);
   t.not(newRule.rule.arn, rule.rule.arn);
+
+  await rules.delete(rule);
+});
+
+test('create a kinesis type rule, using the existing event source mapping', async (t) => {
+  // create two rules with same value
+  const rules = new models.Rule();
+  const newKinesisRule = Object.assign({}, kinesisRule);
+  newKinesisRule.rule = Object.assign({}, kinesisRule.rule);
+  newKinesisRule.name = `${newKinesisRule.name}_new`;
+
+  await rules.create(kinesisRule);
+  const rule = await rules.get({ name: kinesisRule.name });
+
+  await rules.create(newKinesisRule);
+  const newRule = await rules.get({ name: newKinesisRule.name });
+
+  t.not(newRule.name, rule.name);
+  t.is(newRule.rule.value, rule.rule.value);
+  t.false(newRule.rule.arn === undefined);
+  // same event source mapping
+  t.is(newRule.rule.arn, rule.rule.arn);
+
+  await rules.delete(rule);
+  await rules.delete(newRule);
 });
```

Comment:
Not necessarily for this PR but I think we should test our expectations on creating / updating / deleting the lambda event source mappings, I'll add these as todo specs in a separate PR

Reply:
I think these tests covered the event source mapping. e.g.
test('update a kinesis type rule value, resulting in new arn')

Reply:
I see what you are saying but I think we could use a lot more tests here - this is what I am thinking: https://github.com/cumulus-nasa/cumulus/compare/message-updates...abarciauskas-bgse_kinesis-consumer-event-source-mappings-tests - but I wouldn't worry about adding those in this PR!